pymc/sampling/jax.py

```diff
@@ -256,7 +256,7 @@ def _get_batched_jittered_initial_points(
     chains: int,
     initvals: Optional[Union[StartDict, Sequence[Optional[StartDict]]]],
     random_seed: RandomSeed,
-    jitter: bool = True,
+    jitter: Optional[bool] = True,
```

Comment:
The `Optional` hint doesn't make sense. That's for when a variable can have a value of None, but here bool with True/False should suffice

Reply:
Thanks @ricardoV94! Updated